mobile/src/navigations/onboarding-navigator/components/header/styles.ts

```diff
@@ -16,8 +16,8 @@ const styles = StyleSheet.create({
         color: '#fff',
     },
     steps: {
-        width: 65,
-        height: 65,
+        width: 68,
+        height: 68,
```

Comment:
why changing this?

Reply:
To create some space for 'STEP'. These changes are not necessary, but it appears to be very close to breaking without them.

Reply:
maybe some margins are better?

Reply:
I returned them to their previous values; it seems to work without changing them.